question16/style.css

```diff
@@ -80,11 +81,43 @@ td {
 }
 .tabContents ul li a{
     text-decoration: none;
-    color: #3f51b5; 
+    color: #3f51b5;
+    position: relative;
+    line-height: 2;
 }
 .tabContents ul li a:hover{
     border-bottom: 1px solid #3f51b5;
 }
+.tabContents ul li a span.comment_icon_num {
+    background: #e91e63;
+    color: #fff;
+    padding: 0 7px;
+    border-radius: 10px;
+    margin-left: 5px;
+    font-size: 10px;
+    z-index: 1;
+    position: absolute;
+    right: -17px;
+    top: -10px;
+}
+.tabContents ul li a span.new_icon {
```

Comment:
```suggestion
.new_icon {
```

Reply:
I fixed it ! Thank you for poiting it out! 

Latest class: https://github.com/csakyo/homework/commit/e507fac706dd73eee0409c7c71f3f465a3a636ce